it/controllers/pensions/ReliefAtSourceOneOffPaymentsControllerISpec.scala

```diff
@@ -0,0 +1,415 @@
+    val expectedErrorMessage = "Select yes if you made one-off payments into RAS pensions"
+  }
+
+  object ExpectedIndividualCY extends SpecificExpectedResults {
+    val expectedHeading = "Did you make any one-off payments into relief at source (RAS) pensions?"
+    val expectedTitle = "Did you make any one-off payments into relief at source (RAS) pensions?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val thisIncludes: String =
+      s"You told us the total amount you paid plus tax relief was £$someRasAmount. " +
+        "Tell us if this includes any one-off payments. A one-off payment is a single payment, made once."
+    val expectedErrorMessage = "Select yes if you made one-off payments into RAS pensions"
+  }
+
+  object ExpectedAgentEN extends SpecificExpectedResults {
+    val expectedHeading = "Did your client make any one-off payments into relief at source (RAS) pensions?"
+    val expectedTitle = "Did your client make any one-off payments into relief at source (RAS) pensions?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val thisIncludes: String =
+      s"You told us the total amount your client paid plus tax relief was £$someRasAmount. " +
+        "Tell us if this includes any one-off payments. A one-off payment is a single payment, made once."
+    val expectedErrorMessage = "Select yes if your client made one-off payments into RAS pensions"
+  }
+
+  object ExpectedAgentCY extends SpecificExpectedResults {
+    val expectedHeading = "Did your client make any one-off payments into relief at source (RAS) pensions?"
+    val expectedTitle = "Did your client make any one-off payments into relief at source (RAS) pensions?"
+    val expectedErrorTitle = s"Error: $expectedTitle"
+    val thisIncludes: String =
+      s"You told us the total amount your client paid plus tax relief was £$someRasAmount. " +
+        "Tell us if this includes any one-off payments. A one-off payment is a single payment, made once."
+    val expectedErrorMessage = "Select yes if your client made one-off payments into RAS pensions"
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = Seq(
+    UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+    UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+    UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+    UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY))
+  )
+
+  ".show" should {
+    userScenarios.foreach { user =>
+
+      import Selectors._
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+        "render the one-off payments into relief at source (RAS) pensions question page with no pre-filled radio buttons if 'no' CYA question data" which {
```

Comment:
Could you remove the quotes around 'no' in the test name as its a bit misleading, seems like the question data has been set to 'No'